src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -167,11 +175,22 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
             };
 
             // We populate the SelectLists
+            model.Options.TenantsCategories = allSettings
+                .GroupBy(t => t["Category"])
+                .Where(t => !String.IsNullOrEmpty(t.Key))
+                .Select(t => new SelectListItem(t.Key, t.Key))
```

Comment:
here is where you need to add the comparison, otherwise when these items are selected they will never be shown as selected.

Reply:
I didn't get it, I'm just checking if the category is `null` or empty, could you please write a snippet of what you mean

Reply:
```csharp
model.Options.TenantsCategories = allSettings
    .GroupBy(t => t["Category"])
    .Where(t => !String.IsNullOrEmpty(t.Key) && !t.Key.Equals(DefaultCategory, StringComparison.OrdinalIgnoreCase))
    .Select(t => new SelectListItem(t.Key, t.Key))
    .ToList();
```

Is it what you mean?